packages/enzyme-example-mocha/.eslintrc

```diff
@@ -0,0 +1,34 @@
+{
+  "extends": "airbnb",
+  "root": true,
+  "env": {
+    "node": true,
+    "mocha": true
+  },
+  "globals": {
+    "expect": false
+  },
+  "rules": {
+    "id-length": 0,
+    "new-cap": [2, { "capIsNewExceptions": ["AND"] }],
+    "react/jsx-pascal-case": [2, { "allowAllCaps": true }],
+    "react/no-find-dom-node": 1,
+    "import/first": 0,
+    "no-underscore-dangle": [2, {
+      "allowAfterThis": true,
+      "allow": [
```

Comment:
are any of these `allow`s still needed? or this entire rule override?

Reply:
Nope. I removed the rules section.